src/VirtualClient/VirtualClient.Packaging/nuspec/VirtualClient.nuspec

```diff
@@ -27,6 +27,8 @@
         =============================================================================================
         -->
         <file src="$BinConfigDir$\x64\VirtualClient.Main\net6.0\win-x64\publish\**\*" target="content\win-x64" />
+		<file src="$ToolsOutDir$\win-x64\ctstraffic\**\*" target="content\win-x64\packages\ctstraffic\win-x64" />
```

Comment:
Is there a reason we need this in vc package directly?

Reply:
nope, removed.